timeless/restaurants/table_shapes/views.py

```diff
@@ -7,20 +7,20 @@
 from timeless.restaurants import models
 from timeless.restaurants.table_shapes import forms
 
-
 bp = Blueprint("table_shape", __name__, url_prefix="/table_shapes")
 
 
 @bp.route("/")
 def list():
     """List all table shapes
-    @todo #162:30min Implement ordering and sorting of TableShape model for
-     all columns. Update html templates if it's needed. Do not forget write
```

Comment:
@Vatavuk I guess there is a mistake in puzzle ( `ordering and sorting`). Let's leave a new one to implement filters